lib/northbound.c

```diff
@@ -2677,8 +2677,10 @@ const char *nb_client_name(enum nb_client client)
 		return "Pcep";
 	case NB_CLIENT_MGMTD_SERVER:
 		return "MGMTD Server";
-	case NB_CLIENT_MGMTD_BE:
+	case NB_CLIENT_MGMTD_BE:		
```

Comment:
drop white space change

Reply:
click the frrbot in the ci below. You can see the suggestions there. It will also give you a command to apply all of the suggestion by pulling a diff directly to your code.

Reply:
You only have a couple of whitespace changes: https://github.com/FRRouting/frr/pull/18436/checks?check_run_id=39082232753

Reply:
But also see:
```
sprintf, strcat, strcpy, inet_ntoa, ctime are banned; please use snprintf, strlcat, strlcpy, inet_ntop, ctime_r
```